libs/designer/src/lib/ui/styles.less

```diff
@@ -69,6 +69,10 @@
   box-sizing: border-box;
 }
 
+.msla-designer-tools-left-panel {
+  left: 52px !important; // Add extra 32px taken from the collapsed panel
```

Comment:
Is it possible to do this without !important? I really like to make sure we go through every possible alternative before settling on !important

Reply:
If it doesn't take by default, changing the selector to `.msla-designer-tools.msla-designer-tools-left-panel` should cause it to overwrite the base `.msla-designer-tools` styles.

Reply:
Will try that

Reply:
Fixed it